Sources/PrimerSDK/Classes/User Interface/Checkout/VaultCheckoutViewController.swift

```diff
@@ -91,7 +91,7 @@ extension VaultCheckoutViewController: UITableViewDelegate, UITableViewDataSourc
             cell.accessibilityIdentifier = "payment_method_table_view_paypal_cell"
         case .goCardlessMandate:
             cell.accessibilityIdentifier = "payment_method_table_view_direct_debit_cell"
-        case .paymentCard:
+        case .paymentCard, .apaya:
```

Comment:
It should have its own identifier for UI tests, i.e. `payment_method_table_view_apaya`

Reply:
👍  will add